src/lib/connection.ts

```diff
@@ -69,6 +69,12 @@ export interface CreateOptions {
   timeout?: number;
 }
 
+export interface ReadOptions {
+  output?: string;
```

Comment:
Can we have a string enum here so that there's no ambiguity about what the valid options are?